elasticdl_client/README.md

```diff
@@ -0,0 +1,8 @@
+# ElasticDL Client
+
+ElasticDL client is a command-line tool. Users can use it to prepare
+Docker images for a distributed ElasticDL job and submit a train, evaluate
```

Comment:
"Docker images" -> "the Docker image" 
Because a job only needs an image.

Reply:
Good catch, updated.